db/seeds/development.rb

```diff
@@ -40,3 +39,23 @@
                access_token_audience: 'va.gov',
                logout_redirect_uri: 'http://localhost:3001',
                refresh_token_duration: SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES)
+
+# Create Config for example external client using cookie auth
+cookie_template = SignIn::ClientConfig.find_or_initialize_by(client_id: 'cookie_template')
+cookie_template.update!(authentication: SignIn::Constants::Auth::COOKIE,
+                        anti_csrf: true,
+                        redirect_uri: 'http://localhost:4567/auth/result',
+                        access_token_duration: SignIn::Constants::AccessToken::VALIDITY_LENGTH_SHORT_MINUTES,
+                        access_token_audience: 'template',
+                        logout_redirect_uri: 'http://localhost:4567',
+                        refresh_token_duration: SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES)
+
+# Create Config for example external client using api auth
+api_template = SignIn::ClientConfig.find_or_initialize_by(client_id: 'api_template')
+api_template.update!(authentication: SignIn::Constants::Auth::API,
+                     anti_csrf: true,
+                     redirect_uri: 'http://localhost:4567/auth/result',
+                     access_token_duration: SignIn::Constants::AccessToken::VALIDITY_LENGTH_SHORT_MINUTES,
+                     access_token_audience: 'template',
```

Comment:
only need to change this if we change the client id, just a reminder that if we do change the client id we'll want to change the audience to reflect it somehow

Reply:
Yeah, if we named it `sample_client_web/api`, then I think `sample_client` would be appropriate

Reply:
Normally it's a URL, but since the sample client won't be deployed there's no URL (and `localhost` is not a good option)